File: Competitive Programming 3/Section 1.3.3/12250 - Language Detection/Python/main.py
from sys import stdin

wordToLanguage = {
    "HELLO":"ENGLISH",
    "HOLA":"SPANISH",
    "HALLO":"GERMAN",
    "BONJOUR":"FRENCH",
    "CIAO":"ITALIAN",
    "ZDRAVSTVUJTE":"RUSSIAN"
}
def main():
    cs = 1
    for line in stdin:
        if line[0] == '#':
            return
        line = line[:-1]
        print("Case "+str(cs)+": ",end='')
        cs+=1
        if line in wordToLanguage:
            print(wordToLanguage[line])
        else:
            print("UNKNOWN")
main()

File: Competitive Programming 3/Section 1.3.3/12250 - Language Detection/Java/Main.java
import java.util.Scanner;
import java.util.Map;
import java.util.HashMap;

public class Main {
    static Map<String, String> myMap = new HashMap<String, String>() {{
        put("HELLO","ENGLISH");
        put("HOLA","SPANISH");
        put("HALLO","GERMAN");
        put("BONJOUR","FRENCH");
        put("CIAO","ITALIAN");
        put("ZDRAVSTVUJTE","RUSSIAN");      
    }};
    public static void main(String[] args){
        Scanner s = new Scanner(System.in);
        int cs = 1;
        while(s.hasNextLine()){
            String line = s.nextLine();
            if(line.charAt(0)!='#'){
                String value = myMap.get(line);
                System.out.printf("Case %d: %s\n",cs++,value == null ? "UNKNOWN" : value);
            }
        }
    }
}